src/Illuminate/Support/Testing/Fakes/MailFake.php

```diff
@@ -292,6 +292,10 @@ public function send($view, array $data = [], $callback = null)
             return $this->queue($view, $data);
         }
 
+        if (method_exists($view, 'build')) {
+            $view->build();
```

Comment:
As per the comment made on this PR, this needs to be a container call, so that dependencies are resolved.

Reply:
You should also check that not only the method exists, but that it is callable.